src/base.c

```diff
@@ -377,10 +377,11 @@ int cw_authenticate(const char *service, const char *username, const char *passw
 	};
 
 	rc = pam_start(service, creds.username, &convo, &pam);
-	if (rc == PAM_SUCCESS)
+	if (rc == PAM_SUCCESS) {
 		rc = pam_authenticate(pam, PAM_DISALLOW_NULL_AUTHTOK);
 		if (rc == PAM_SUCCESS)
 			rc = pam_acct_mgmt(pam, PAM_DISALLOW_NULL_AUTHTOK);
+	}
```

Comment:
I think i'd rather see this as the following (possible white-spaced cleaned-up) steps:

```
rc = pam_start(...);
if (rc == PAM_SUCCESS)
  rc = pam_authenticate(...);
if (rc == PAM_SUCCESS)
  rc = pam_acct_mgmt(...);
```

yeah, it's double the rc check, but i think it reads better.